pode.build.ps1

```diff
@@ -454,6 +454,41 @@ Task InstallCurrentUser  {
         Copy-Item -Path $(Join-Path -Path $path -ChildPath 'LICENSE.txt') -Destination $dest -Force   | Out-Null
 
         Write-Host "Deployed to $dest"
+    }else{
+        Write-Error "Parameter -Version is required"
     }
 
+}
+
+
+Task Sign{
```

Comment:
I like this idea of signing the Pode module, I've thought about it in the past. However this should be it's own workstream/ticket, not a part of OpenAPI. Please remove and open a dedicated ticket/PR so it can be investigated separately.

Reply:
it's not called from anywhere.
it's a standalone task
I added the signing for my internal build I cannot put anything in production that is not signed.

to Sign you have to use `invoke-Build Pack -Version 2.10.0` folowed by `invoke-Build pack -Version 2.10.0`
later you can use `invoke-Build InstallCurrentUser -Version 2.10.0` to install the bundle under current user

Reply:
Be careful about adding things that are custom to only you, they should really be coded into your own deployment pipelines. It would be better for them to be there, rather than here in Pode unless it's official logic for all.

Reply:
it's not custom it's work from everyone. but I can understand your point of view 
Let me remove it from the build and I'm going to open a new feature request for this